addon/components/besluit-plugin-card.js

```diff
@@ -24,6 +25,37 @@ export default class EditorPluginsTemplateVariableCardComponent extends Componen
     this.args.controller.executeCommand('insert-title', this.args.controller);
   }
 
+  @action
+  deleteArticle() {
+    const range = this.args.controller.rangeFactory.fromAroundNode(
+      this.articleElement
+    );
+    this.args.controller.selection.selectRange(range);
+    this.args.controller.executeCommand('delete-selection');
+  }
+
+  @action
+  moveUpArticle() {
+    this.args.controller.executeCommand(
+      'move-article',
+      this.args.controller,
+      this.besluitUri,
+      this.articleElement,
+      true
+    );
+  }
+
+  @action
+  moveDownArticle() {
+    this.args.controller.executeCommand(
+      'move-article',
+      this.args.controller,
+      this.besluitUri,
+      this.articleElement,
+      false
+    );
+  }
+
   @action
   modelWrittenHandler() {
```

Comment:
this handler should trigger on selectionChange, not contentChange, and should be renamed accordingly
you might hit the same bug you are hitting in the besluitType plugin if you do this, I'll get that sorted asap

Reply:
Yeah, I didn't want to use the new version of the editor as it will run into the same bug, when you fix that I will upgrade and change the event :)